keewe-api/src/main/java/ccc/keeweapi/api/insight/InsightController.java

```diff
@@ -30,6 +33,15 @@ public ApiResponse<InsightGetResponse> getInsight(@PathVariable Long insightId)
         return ApiResponse.ok(insightApiService.getInsight(insightId));
     }
 
+    @GetMapping
+    public ApiResponse<List<InsightGetForHomeResponse>> getInsightsForHome(
+            @RequestParam Long cursor,
+            @RequestParam Long limit,
+            @RequestParam @Nullable Boolean follow
```

Comment:
레포지토리에서 null check를 하기보단, 이렇게 기본 값을 설정해주는 방법은 어떨까요?
```suggestion
            @RequestParam(required = false, defaultValue = "true") follow
```